weave/trace_server/migrations/005_add_cost.up.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE llm_token_prices (
+    /*
+    `pricing_level`: The level at which the token pricing applies. It can be 'default' for global pricing,
+    'project' for project-specific pricing, or 'org' for organization-specific pricing.
+    */
+    pricing_level String,
+
+    /*
+    `filter_id`: The filter identifier for the token pricing. It can be 'default' or any other string. This allow with type to uniquely identify where the pricing applies.
+    */
+    filter_id String,
+
```

Comment:
might want to consider adding a provider (think about the case where the provider has different pricing for the same model id)